playground/mocks/idp/idx/data/factor-enroll-options.json

```diff
@@ -12,58 +12,61 @@
           "create-form"
         ],
         "name": "select-factor",
-        "href": "http://localhost:3000/idp/idx",
+        "href": "http://manumalhotra.okta1.com:1802/idp/idx/credential/enroll",
         "method": "POST",
+        "accepts": "application/vnd.okta.v1+json",
         "value": [
           {
-            "name": "factorType",
+            "name": "factorProfileId",
             "type": "set",
             "options": [
               {
-                "label": "Password",
-                "value": "password"
-              },
-              {
-                "label": "E-mail",
-                "value": "email"
+                "label": "WebAuthn_xAtmO",
+                "value": "fpr1tqz7XQcy7LGN40g4"
               }
             ]
           },
           {
             "name": "stateHandle",
-            "value": "01OCl7uyAUC4CUqHsObI9bvFiq01cRFgbnpJQ1bz82",
-            "visible": false
+            "required": true,
+            "value": "02CqFbzJ_zMGCqXut-1CNXfafiTkh9wGlbFqi9Xupt",
+            "visible": false,
+            "mutable": false
           }
         ]
       }
     ]
   },
+  "factors": {
+    "type": "array",
+    "value": [
+      {
+        "factorType": "webauthn",
+        "factorProfileId": "fpr1tqz7XQcy7LGN40g4"
+      }
+    ]
+  },
   "user": {
     "type": "object",
     "value": {
-      "id": "I9bvFiq01cRFgbn",
-      "passwordChanged": "2019-05-03T19:00:00.000Z",
-      "profile": {
```

Comment:
hmm, the latest API response got `profile` removed?

Reply:
IDK this is what is returned for webauthn. SIW dint wok(UI  dint render)  with the old response that was there in the file and I had to update the json. Can u guys give me an actual email enroll response ?

Reply:
Updated with email and password itself.